jellyfin_kodi/player.py

```diff
@@ -147,18 +148,68 @@ def set_item(self, file, item):
         volume = result.get('volume')
         muted = result.get('muted')
 
+        jelly_client = Jellyfin(item['ServerId']).get_client()
+        current_item = jelly_client.jellyfin.get_item(item['Id'])
+        edl = []
+        
+        try:
+            edl = self.get_edl_content(current_item['Path'])
+        except :
+            edl = []
+
         item.update({
             'File': file,
             'CurrentPosition': item.get('CurrentPosition') or int(seektime),
             'Muted': muted,
             'Volume': volume,
-            'Server': Jellyfin(item['ServerId']).get_client(),
-            'Paused': False
+            'Server': jelly_client,
+            'Paused': False,
+            'EdlContent': edl
         })
 
         self.played[file] = item
         LOG.info("-->[ play/%s ] %s", item['Id'], item)
 
+        event("ItemEDLLoaded", edl, hexlify=False)
```

Comment:
publish event here.
i want to process the skip or promote action in a clean addon, not here.
so I need to publish edl data